styles.css

```diff
@@ -0,0 +1,316 @@
+@media (min-width: 375px) and (max-width: 768px) {
+  body {
+    margin: 0;
+  }
+
+  #header {
+    height: 4.625rem;
+  }
+
+  .Menu {
+    display: flex;
+    flex-direction: column;
+    align-items: flex-end;
+  }
+
+  .Menu-line {
+    width: 1.25rem;
+    height: 0.125rem;
+    background-color: black;
+    margin: 0.25rem 0;
+  }
+
+  h1 {
+    font-size: 3.125rem;
+    font-family: 'Playfair Display', serif;
+    line-height: 100%;
+  }
+
+  #headline h1 {
+    margin: 0 1rem;
+    word-spacing: 30rem;
+  }
+
+  p {
+    color: #1a2236;
+    line-height: 1.313rem;
+    font-family: 'Poppins', sans-serif;
+    font-weight: 400;
+  }
+
+  .Intro p {
+    font-family: 'Poppins', sans-serif;
+    font-size: 0.875rem;
+    line-height: 1.5rem;
+    margin-left: 1rem;
+  }
+
+  .image1 {
+    position: absolute;
+    left: 0;
+    top: 0;
+  }
+
+  .image2 {
+    position: absolute;
+    left: 2.188rem;
+    top: 0;
+  }
+
+  .Intro {
+    display: flex;
+    flex-direction: column;
+  }
+
+  .image3 {
+    margin: 0;
+  }
+
+  .Collaboration {
+    background-color: #1a2236;
+    color: white;
+    height: 3.125rem;
+    font-size: 1rem;
+    box-sizing: border-box;
+    margin: 1rem 1rem;
+  }
+
+  .Projects {
+    padding: 2rem 1rem;
+  }
+
+  .work {
+    display: grid;
+    grid-template-rows: 21.375rem 21.375rem 21.375rem 21.375rem 21.375rem 21.375rem;
+  }
+
+  .work div {
+    background-color: #1a2236;
+    padding: 0;
+  }
+
+  .work img {
+    height: 11.625rem;
+    width: 100%;
+  }
+
+  h3 {
+    color: white;
+    line-height: 80%;
+    font-family: 'Poppins', sans-serif;
+    font-weight: 300;
+    margin-bottom: 0;
+  }
+
+  .tools {
+    display: flex;
+    justify-content: flex-start;
+    gap: 0.625rem;
+    list-style-type: none;
+    padding: 0;
+  }
+
+  .tools li {
+    background-color: #98dede;
+    padding: 0.375rem 0.625rem;
+    font-family: 'Poppins', sans-serif;
+    font-weight: 300;
+  }
+
+  .projects {
+    margin-left: 1.25rem;
+    display: flex;
+    flex-direction: column;
+  }
+
+  .projects button {
+    color: white;
+    border: 1px solid white;
+    background-color: #1a2236;
+    width: 11.125rem;
+    height: 2.75rem;
+    line-height: 1.5rem;
+    font-family: 'Poppins', sans-serif;
+    font-weight: 500;
+  }
```

Comment:
- [ ] Please I kindly suggest adding a `hover` effect to the `.projects` `button`. 😄